build_tools/scripts/bazel_to_cmake.py

```diff
@@ -146,6 +173,12 @@ def _convert_target(self, target):
         target = package  # Omit target if it matches the package name
       else:
         target = package + ":" + target
+      if target.endswith("_gen"):
+        # Files created by gentbl have to be included as source and header files
+        # and not as a dependency. Adding these targets to the dependencies list,
+        # results in linkage failures if the library including the gentbl dep is
+        # marked as ALWAYSLINK.
+        return
```

Comment:
Is returning `None` appropriate here, or should this be empty string `""`?

Either way, the `deps_list` handling below that calls this function should be a bit more explicit about how it handles targets that do not convert. That could be handled in either the list comprehension step or in the de-dup step. For example:
```
deps_list = []
for dep in deps:
  converted_dep = self._convert_target(dep)
  if dep:
    deps_list.append(dep)
```

```
deps_list = [self._convert_target(dep) for dep in deps]
# Remove `None` values and duplicates, preserving the original ordering.
deps_list = ...
```

Reply:
It's a matter of taste, but it was intended to return `None` and not an empty string. The latter would also be fine and does not even require a require a different filter. Changed it to `""`, since is seems to be more consistent.